hazelcast/src/test/java/com/hazelcast/test/starter/constructor/EntryEventConstructor.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.test.starter.constructor;
+
+import com.hazelcast.test.starter.HazelcastStarterConstructor;
+
+import static com.hazelcast.test.starter.HazelcastProxyFactory.ProxyPolicy.SUBCLASS_PROXY;
+
+@HazelcastStarterConstructor(classNames = {"com.hazelcast.core.EntryEvent"}, proxyPolicy = SUBCLASS_PROXY)
```

Comment:
Example usage for a `SUBCLASS_PROXY` class. As I said, I like this more than adding this at two places in the factory.